cpp/src/parquet/column_reader.cc

```diff
@@ -2088,6 +2110,10 @@ class FLBARecordReader final : public TypedRecordReader<FLBAType>,
   }
 
   void ReadValuesSpaced(int64_t values_to_read, int64_t null_count) override {
+    if (null_count == 0) {
+      ReadValuesDense(values_to_read);
```

Comment:
I've checked `FLBARecordReader`, it's ReadDense uses a extra `null_bitmap_builder_`, sigh